index.html

```diff
@@ -6883,6 +7140,303 @@ <h2 id="MediaStreamAudioSourceNode">
         </dl>
       </section>
     </section>
+    <section>
+      <h2> Processing model </h2>
+      <section class="informative">
+      <h3> Background </h3>
+        <p>
+          Real-time audio systems that require low latency are, often time,
+          implemented using <em>callback functions</em>, where the operating
+          system calls the program back when more audio has to be computed in
+          order for the playback to stay uninterrupted. Such callback is called
+          on a high priority thread (often the highest priority on the system).
+          This means that a program that deals with audio only executes code
+          from this callback, as any buffering between a rendering thread and
+          the callback would naturally add latency or make the system less
+          resilient to glitches.
+        </p>
+        <p>
+          For this reason, the traditional way of executing asynchronous
+          operations on the Web Platform, the event loop, does not work here,
+          as the thread is no <em>continuously executing</em>. Additionaly, a
+          lot of unnecessary and potentially blocking operations are available
+          from traditional execution contexts (Windows and Workers), which is
+          not something that is desirable to reach an acceptable level of
+          performance.
+        </p>
+        <p>
+          Additionaly, the Worker model makes creating a dedicated thread
+          necessary for a script execution context, while all <a>AudioNode</a>s
+          usually share the same execution context.
+        </p>
+      </section>
+      <section>
+        <h3> Control thread and rendering thread </h3>
+        <p>
+          The Web Audio API MUST be implemented using a <a>control thread</a>,
+          and one <a>rendering thread</a>.
+        </p>
+        <p>
+          The <dfn>control thread</dfn> is the thread from
+          which the <a>AudioContext</a> is instanciated, and from which authors
+          manipulate the audio graph. The <dfn>rendering
```

Comment:
It would be helpful if there was more of a identification between the term _control thread_ and the thread from which an operation is being invoked. You do say the thread "...from which authors manipulate the audio graph" but would it be clearer to identify the control thread with the thread in which page-level scripts run?